apps/dialtone-documentation/docs/.vuepress/views/IconCatalog.vue

```diff
@@ -186,14 +197,47 @@ const selectIcon = (icon) => {
   else isPopoverOpen.value[icon.name] = !isPopoverOpen.value[icon.name];
 };
 
+const scrollToIcon = (iconName) => {
+  if (isMobile.value) {
+    isModalOpen.value = true;
+  } else {
+    isPopoverOpen.value[iconName] = !isPopoverOpen.value[iconName];
+    nextTick(() => {
+      const iconElement = document.getElementById(`in-${iconName}`); // Scroll to the opened popover
+      if (iconElement) {
+        // there is a style in dialtone-docs.less setting scroll-behavior: smooth, so
+        // we need to first set it to auto to avoid having the scroll animation
+        document.documentElement.style.scrollBehavior = 'auto';
+        iconElement.scrollIntoView({ behavior: 'instant' });
+        // set the scroll behavior to smooth again
+        // eslint-disable-next-line no-return-assign
+        setTimeout(() => document.documentElement.style.scrollBehavior = 'smooth', 5);
```

Comment:
We shouldn't set timeouts like this to wait for things to complete because they might need to be different depending on the speed of your system.

There is a scroll-end event that I think would work here. https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollend_event

Reply:
You are totally right, fixed in the last commit